examples/super_resolution/config.json

```diff
@@ -0,0 +1,44 @@
+{
+    "verbose": true,
+    "input_model":{
+        "type": "PyTorchModel",
+        "config": {
+            "name": "supres",
+            "model_path": null,
+            "is_file": false,
+            "model_loader": "load_pytorch_model",
+            "model_script": "examples/supres/loader.py"
```

Comment:
can you change the path to just `loader.py`? All paths in our examples are relative to the directory each example is in.

Reply:
done